google/cloud/pubsub_v1/types.py

```diff
@@ -112,6 +114,10 @@
         "The maximum amount of time in seconds to hold a lease on a message "
         "before dropping it from the lease management."
     )
+    FlowControl.max_extension_period.__doc__ = (
+        "Bounds the maximum amount of time before a message redelivery in the "
+        "event the subscriber fails to extend the deadline."
+    )
```

Comment:
Can we here mention the unit of time like in `seconds` or in `millisecond` as mentioned in other doc

Reply:
Sure